### PR TITLE
Fix merge inconsistencies related to slf4j

### PR DESCRIPTION
**Pull request by dsKarthick**
Yesterday's scheduler merge resulted in some slf4j related inconsistencies. This PR aims to fix that.

### CODE REVIEW

**Comment by JessicaLHartog** [comment]
:ship: :it: LGTM

File: storm/src/main/storm/mesos/schedulers/SchedulerUtils.java [hunk 1]
```diff
@@ -101,21 +102,20 @@ public static MesosWorkerSlot createWorkerSlotFromOfferResources(Map mesosStormC
...
-               "OfferedCpu: " + offerResources.getCpu() + " at node: " + offerResources.getHostName());
+      log.warn("Refusing to create worker slot. requestedWorkerCpu: {} but OfferedCpu: {} at node: {}",
+               requestedWorkerCpu, offerResources.getCpu(), offerResources.getHostName());
       return null;
     }
 
     if (requestedWorkerMem > offerResources.getMem()) {
-      log.warn("Refusing to create worker slot. requestedWorkerMem: " + requestedWorkerMem + " but " +
-               "OfferedMem: " + offerResources.getMem() + " at node: " + offerResources.getHostName());
+      log.warn("Refusing to create worker slot. requestedWorkerMem: {} but OfferedMem: {} at node: {}", requestedWorkerMem, offerResources.getMem(), offerResources.getHostName());
```
Comment (erikdw) [inline storm/src/main/storm/mesos/schedulers/SchedulerUtils.java]: wrap the line?

File: storm/src/main/storm/mesos/schedulers/SchedulerUtils.java [hunk 2]
```diff
@@ -87,7 +88,7 @@ public static boolean supervisorExists(String offerHost, Collection<SupervisorDe
         offerResourcesListPerNode.put(hostName, new ArrayList<OfferResources>());
       }
       offerResourcesListPerNode.get(hostName).add(offerResources);
-      log.info("Available resources at " + hostName + ": " + offerResources.toString());
+      log.info("Available resources at {} : {}", hostName, offerResources.toString());
```
Comment (erikdw) [inline storm/src/main/storm/mesos/schedulers/SchedulerUtils.java]: nit: this changed the format from:  `host.com: My Resources` to `host.com : My Resources`.  Not sure it matters.

**Comment by erikdw** [comment]
super minor nits.  Address at your pleasure and squash and merge.  :ship:

**Pull request merged**